server/index.js

```diff
@@ -9,6 +9,7 @@ app.use(express.static(__dirname + "/../public"));
 
 app.use('/api/overall_reviews', overallReviews);
 
+
 app.get('/api/individual_reviews/1' , (req, res) => {
   Review.find({}, { user: 1, imageURL: 1, date: 1, reviewTxt: 1, _id: 0}, (err, results) => {
     if (err) {
```

Comment:
Make sure we don't send back errors that are generated by the database. We don't want to give a user any information about the technology we are using on the backend. Instead we can send a 500 code and a custom message that we have defined.